components/pedigree/resources/src/main/resources/pedigree/view/personGroupVisuals.js

```diff
@@ -26,14 +26,6 @@ define([
             this.setNumPersons(node.getNumPersons());
         },
 
-        generateHoverbox: function(x, y) {
-            if (editor.isReadOnlyMode()) {
-                return new ReadOnlyHoverbox(this.getNode(), x, y, this.getGenderGraphics());
-            } else {
-                return new PersonGroupHoverbox(this.getNode(), x, y, this.getGenderGraphics());
```

Comment:
it was related to the issue as all your changes to PersonGroupHoverbox were negated by the commit (JFYI - the hoverbox type for a node is decided by the method you removed, so after the change group nodes would inherit the parent method, which generates a PersonHoverbox, which has stuff not relevant to groups). Now looks good